src/Alpha_complex/include/gudhi/Alpha_complex.h

```diff
@@ -62,6 +62,10 @@ namespace alpha_complex {
 template<typename D> struct Is_Epeck_D { static const bool value = false; };
 template<typename D> struct Is_Epeck_D<CGAL::Epeck_d<D>> { static const bool value = true; };
 
+template<typename D> struct Is_Dynamic_dimension_tag { static const bool value = false; };
+template<> struct Is_Dynamic_dimension_tag<CGAL::Epeck_d<CGAL::Dynamic_dimension_tag>> { static const bool value = true; };
+template<> struct Is_Dynamic_dimension_tag<CGAL::Epick_d<CGAL::Dynamic_dimension_tag>> { static const bool value = true; };
```

Comment:
We could also use `std::is_same_v<typename Kernel::Dimension, CGAL::Dynamic_dimension_tag>` to be slightly more generic, instead of listing the 2 cases Epick_d and Epeck_d.

Reply:
Done on [13f2944](https://github.com/GUDHI/gudhi-devel/pull/1050/commits/13f2944b2d17849310c567a35a5d0fd096b40f62)